site/docs/v1/tech/apis/_webauthn-response-body-base.adoc

```diff
@@ -0,0 +1,77 @@
+[api]
+ifeval::["{base_field_name}" != "credential"]
+[field]#{base_field_name}# [type]#[Array]#::
+The list of WebAuthn Passkey objects.
+endif::[]
+
+[field]#{base_field_name}.algorithm# [type]#[Long]#::
+The identifier for the signing algorithm used with the passkey. These values are defined by https://www.rfc-editor.org/rfc/rfc9053[RFC 9053]. FusionAuth supports a subset of these algorithms.
+
+[field]#{base_field_name}.attestationType# [type]#[String]#::
+The type of attestation provided when the passkey was registered. Supported values include:
```

Comment:
For the response, you don't need to provide 'supported value' sections.

I'd change this to 
```suggestion
The type of attestation provided when the passkey was registered.
```